frontend/src/game/base/Game.ts

```diff
@@ -22,10 +22,12 @@ export class Game {
   protected camera: Camera;
   protected map: Map;
   protected player: Player;
+  public your_menu;
```

Comment:
Make it consistent, change to yourMenu if it doesn't affect many files outside this one

Reply:
esta a ser usado depois para atualizar as notificoes